### PR TITLE
 Update CSV, separate first and last names, add instructor

### PR DESCRIPTION
**Pull request by gotonode**
### Short description

implement #948
### DoD

I have:
- [x] added actual code.
- [x] produced clean code that passes ESLint.
- [x] code that actually does what it should.
- [ ] documented the code or added documentation to the wiki.
- [x] added or modified test(s).
- [x] test(s) that actually pass.

### CODE REVIEW

**Comment by gotonode** [comment]
Works otherwise but the instructor field is currently just a mock value. Feel free to make the necessary change to the const and add a ready-for-pull message. This issue has topmost priority.

**Comment by gotonode** [comment]
I'm test-driving a new label called "top priority" which should be a call to action on that particular issue ASAP. Once it's been completed, remove that label and assign it to the next topmost priority issue.

File: labtool2.0/src/tests/CoursePage.test.js [hunk 1]
```diff
@@ -320,7 +320,12 @@ describe('<CoursePage /> as teacher', () => {
         .find({ children: 'Export CSV of all students' })
         .simulate('click')
 
-      expect(mockDownloadFile).toBeCalledWith(expect.anything(), expect.anything(), expect.stringContaining('Maarit Mirja Opiskelija,014578343,maarit.opiskelija@helsinki.invalid,Tiran labraprojekti'))
+      // TODO: Remember to add the instructor to the following line once it has been implemented!
```
Comment (hisahi) [inline labtool2.0/src/tests/CoursePage.test.js]: This comment should not be here anymore

**Comment by gotonode** [comment]
Now it looks good.

**Pull request merged**